DESCRIPTION

```diff
@@ -1,12 +1,42 @@
 Package: datadelay
 Title: Delay Functions for Epidemic Data
 Version: 0.1
-Authors@R: 
-    person("Adam", "Kucharski", , "adam.kucharski@lshtm.ac.uk", role = c("aut", "cre"),
-           comment = c(ORCID = "0000-0001-8814-9421"))
+Authors@R: c(
+    person(
+        given = "Adam",
+        family = "Kucharski",
+        email = "adam.kucharski@lshtm.ac.uk", 
+        role = c("aut", "cre"),
+        comment = c(ORCID = "0000-0001-8814-9421")
+        ),
+    person(
+        given = "Tim",
+        family = "Russell",
+        email = "timothy.russell@lshtm.ac.uk", 
+        role = c("aut"),
+        comment = c(ORCID = "0000-0001-5610-6080")
+        )
+    )
 Description: Delay functions for routine surveillance data.
-License: `use_mit_license()`, `use_gpl3_license()` or friends to pick a
-    license
+License: MIT + file LICENSE
+URL: https://github.com/epiverse-trace/datadelay
+BugReports: https://github.com/epiverse-trace/datadelay/issues
 Encoding: UTF-8
 Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.1.2
+RoxygenNote: 7.2.2
+Imports:
+    data.table,
+    ggplot2,
+    patchwork
+Suggests: 
+    testthat (>= 3.0.0),
+    epiparameter,
+    usethis,
```

Comment:
No need to put usethis here as it's a dev package. Same way we don't put roxygen2